<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>741</width>
    <height>465</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QStackedWidget" name="stackedProjectsWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="selectProjectWidget">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="selectProjectLabel">
          <property name="text">
           <string>Bitte wählen Sie ein Projekt aus dem Projekt-Menü aus.</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>741</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="title">
     <string>Datei</string>
    </property>
    <widget class="QMenu" name="menuOpenDevice">
     <property name="title">
      <string>Kamera auswählen</string>
     </property>
    </widget>
    <addaction name="actionOpenFile"/>
    <addaction name="menuOpenDevice"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuProjects">
    <property name="title">
     <string>Projekt</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuProjects"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpenFile">
   <property name="text">
    <string>Datei öffnen</string>
   </property>
  </action>
  <action name="actionLas">
   <property name="text">
    <string>las</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Beenden</string>
   </property>
  </action>
  <action name="actionProjekt_A">
   <property name="text">
    <string>Projekt A</string>
   </property>
  </action>
  <action name="actionProjekt_B">
   <property name="text">
    <string>Projekt B</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
